### PR TITLE
Support `log-path` opt for json-file logging driver

### PR DESCRIPTION
**Pull request by Bingmang**
Support specifying log-path when using json-file logging driver.

Signed-off-by: Bingmang <todd.g@qq.com>

### CODE REVIEW

**Comment by fahedouch** [comment]
could you please add a unit test.

**Review by fahedouch — COMMENTED** [review]
could you please add a unit test.

File: cmd/nerdctl/run_test.go [hunk 1]
```diff
@@ -248,6 +248,40 @@ func TestRunWithJsonFileLogDriver(t *testing.T) {
...
 
+func TestRunWithJsonFileLogDriverAndLogPathOpt(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("json-file log driver is not yet implemented on Windows")
+	}
+	base := testutil.NewBase(t)
+	containerName := testutil.Identifier(t)
+
+	defer base.Cmd("rm", "-f", containerName).AssertOK()
+	customLogJSONPath := filepath.Join(os.TempDir(), containerName, containerName+"-json.log")
```
Comment (AkihiroSuda) [inline cmd/nerdctl/run_test.go:259]: Please use `t.TempDir()`

**Comment by Bingmang** [comment]
The test seems to be unstable. :(

**Review by fahedouch — APPROVED** [review]

**Comment by fahedouch** [comment]
Thanks

**Comment by Bingmang** [comment]
Thanks for reviewing too

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**